### PR TITLE
Better dbt seeding in tests

### PR DESCRIPTION
**Pull request by turbo1912**

### CODE REVIEW

File: integration_tests/features/environment.py [hunk 1]
```diff
@@ -13,4 +14,5 @@ def after_all(context):
 
 
 def after_scenario(context, scenario):
-    os.system("rm -rf mock/temp/*")
+    if not hasattr(context, "temp_dir"):
```
Comment (chamini2) [inline integration_tests/features/environment.py:17]: should this be `if hasattr`?

File: integration_tests/projects/001_flow_run_with_selectors/data/time_series.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-id,y,ds
```
Comment (chamini2) [inline integration_tests/projects/001_flow_run_with_selectors/data/time_series.csv:1]: why?

File: integration_tests/features/environment.py [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 
 def before_all(context):
-    os.system("mkdir mock/temp")
-    os.system("dbt seed --profiles-dir mock --project-dir mock")
+    if not hasattr(context, "temp_dir"):
+        os.system("mkdir mock/temp")
+        os.system("dbt seed --profiles-dir mock --project-dir mock")
```
[the remark this answers is not in the document]
Comment (turbo1912) [reply integration_tests/features/environment.py:7]: yea we dont want any of these things in the `environment.py` but we need them for the older tests. .

File: integration_tests/features/environment.py (hunk 1 above)
Comment (turbo1912) [reply integration_tests/features/environment.py:17]: so this is an indicator if the tests run with the new style or the old style, we dont want to do this if it ran with the new style..

**Pull request merged**